cmd/dp-dataset-exporter/main.go

```diff
@@ -24,10 +24,33 @@ import (
 	"github.com/ONSdigital/dp-dataset-exporter/schema"
 )
 
+const (
+	serviceConsumer            = "kafka-consumer"
+	serviceCsvExportedProducer = "kafka-csv-exported-producer"
+	serviceErrorProducer       = "kafka-error-producer"
+	serviceFileStore           = "file-store"
+	serviceGraph               = "graph"
+	serviceHealthTicker        = "health-ticker"
+	serviceVault               = "vault"
+)
+
+var services = map[string]bool{
```

Comment:
I think a comment here to say its a map of service name to init status would be useful for anyone glancing through the code.